Server 1633 (#6462)
circleci/circleci-docs: jekyll/_cci2/server-3-install.adoc

.Frontend Settings
image::server-3-frontend-settings.png[Screenshot showing frontend settings]

Complete the fields described below. You can either supply a private key and certificate, or check the box that allows Let's Encrypt or https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html[AWS Certificate Manager (ACM)] to automatically request and manage certificates for you. You can also disable TLS termination at this point, but the system will still need to be accessed over HTTPS.

==== Using ACM TLS Certificates

> If you'd like to use https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html[AWS Certificate Manager (ACM)] to manage your TLS certificates. you can follow the https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html[ACM documentation] for instructions on how to generate ACM certificates.

Reviewer: ```suggestion
If you would like to use https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html[AWS Certificate Manager (ACM)] to manage your TLS certificates, you can follow the https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html[ACM documentation] for instructions on how to generate ACM certificates.
```

> Once your have generated your certificates, you can enable ACM in your KOTS admin console under the Frontend section. You only need to check the ACM box, provide your ACM ARN (Amazon Resource Name) as in the example below.

Reviewer: ```suggestion
Once you have generated your certificates, you can enable ACM from the KOTS admin console under the Frontend section. Check the ACM box and provide your ACM ARN (Amazon Resource Name), as shown in the example below.
```
